generate.sh

```diff
@@ -182,6 +183,11 @@ if [ -n "$APIDOCS_NAME" ] && [ -n "$APIDOCS_URL" ]; then
   fi
 fi
 
+if [[  -n "$JAVADOC_PATH" ]]; then
+  javadoc -sourcepath $BASE/temp/$EMAIL/$UNIQUEID/$JAVADOC_PATH -d $BASE/temp/$EMAIL/$UNIQUEID/javadoc -subpackages . >> ${LOGFILE} 2>>${LOGFILE}
```

Comment:
You can replace `$BASE/temp/$EMAIL/$UNIQUEID` with `$ROOT_DIR`.
Also instead of generating javadocs there and then moving to `_build/html`, can we directly specify `$BUILD_DIR/_build/html/javadoc` as the destination?